python/Makefile

```
@@ -2,6 +2,7 @@
 
 VENV := venv
 MATURIN_VERSION := $(shell awk -F '[ ="]+' '$$1 == "requires" { print $$4 }' pyproject.toml)
+TARGET_WHL := $(wildcard ../target/wheels/*.whl)
```

We could retrieve the version of the `Cargo.toml` file using the command`cargo pkgid`.

 ```suggestion
PACKAGE_VERSION := $(shell cargo pkgid | cut -d\# -f2 | cut -d: -f2)
TARGET_WHEEL := $(shell ls ../target/wheels/deltalake-${PACKAGE_VERSION}-*.whl)
```